java/code/src/com/redhat/rhn/domain/org/test/OrgFactoryTest.java

```diff
@@ -187,12 +187,10 @@ public void testOrgDefautRegistrationToken() throws Exception {
         orig.setToken(token);
         orig = OrgFactory.save(orig);
         Long origId = orig.getId();
-        flushAndEvict(orig);
```

Comment:
not sure if it is a good idea to remove this.
`flushAndEvict` take care that hibernate write its cache to DB and remove all these objects from the cache.

If we remove this, the next calls might take the cached objects and not in DB yet.

If this cause failed unit tests, we may have bug in the mapping

Reply:
I re-added it, tests looks fine.